src/Hl7.Fhir.Support.Poco/Model/FhirTypeConstants.cs

```diff
@@ -0,0 +1,51 @@
+﻿/* 
+ * Copyright (c) 2021, Firely (info@fire.ly) and contributors
+ * See the file CONTRIBUTORS for details.
+ * 
+ * This file is licensed under the BSD 3-Clause license
+ * available at https://github.com/FirelyTeam/firely-net-sdk/blob/master/LICENSE
+ */
+
+namespace Hl7.Fhir.Support.Poco
+{
+    /// <summary>
+    /// List of Fhir types constants. This will be removed when we introduce ModelSpace, but for now we will use this.
+    /// </summary>
+    internal class FhirTypeConstants
```

Comment:
`internal` - this means other assemblies have to re-do this work. On the other hand, if you make this public, "why didn't you add a constant for X"  (because *we* didn't need it ;-))......

Reply:
Internal, because when we introduce ModelSpace we can safely remove this class without breaking the code.